.circleci/config.yml

```diff
@@ -8,7 +8,7 @@ jobs:
         environment:
           - PGHOST: 127.0.0.1
           - PGUSER: postgres
-      - image: circleci/postgres:9.6
+      - image: circleci/postgres:10.5
```

Comment:
ez-rails Circle config uses the postgres:9.4.14-alpine-postgis-ram image (which has the postgis extension). I don't think it matters too much for this gem, unless we want to keep the db image in sync with what's used in ez-rails, but I thought I'd point it out.

Reply:
I think this is fine.